### PR TITLE
Story 5

### PR DESCRIPTION
**Pull request by agallant121**
Complete user story 5

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@ def current_merchant
     current_user && current_user.merchant? if session[:user_id]
   end
 
+  def current_admin
```
Comment (agallant121) [inline app/controllers/application_controller.rb:19]: Add current admin helper method

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@ def create
     end
     if user.merchant?
       redirect_to '/merchant/dashboard'
+    elsif user.admin?
```
Comment (agallant121) [inline app/controllers/sessions_controller.rb:14]: Add admin conditional for session creation

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -14,7 +14,11 @@
       <%= link_to "Home", "/"%>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
+      <%= link_to "Cart: #{cart.total_items}", "/cart" unless current_admin %>
```
Comment (agallant121) [inline app/views/layouts/application.html.erb:17]: Add conditional for admin nav bar views

File: spec/features/navigation/navigation_spec.rb [hunk 4]
```diff
@@ -76,8 +76,48 @@
...
+     )
+
+    visit '/login'
+
+    fill_in :email, with: "jaffar@gamil.com"
+    fill_in :password, with: "geniessuck"
+
+    click_button "Log In"
+save_and_open_page
+    expect(page).to have_link("Admin Dashboard")
```
Comment (agallant121) [inline spec/features/navigation/navigation_spec.rb:100]: Admin nav bar spec looking for certain links to show and not show up according to story 5

**Comment by palworth** [comment]
Story 5 complete. Ready for master.

**Pull request merged**